scripts/job_bq_webdataset.sh

```
@@ -0,0 +1,144 @@
+
+NVME="${SLURM_TMPDIR}"
+OUTPUT_DIR="/scratch/melabbas/global_wds"
+SPLITS_DIR="/project/6068129/melabbas/ami-ml/data/splits"   # contains train.csv val.csv test.csv
+IMAGES_PER_SHARD=1000
+BATCH_SIZE=5
+PACK_WORKERS=16
+
+echo "=== build_wds_global started at $(date) ==="
+echo "Node: $(hostname)"
+echo "NVMe: ${NVME}  ($(df -h ${NVME} | tail -1 | awk '{print $4}') free)"
+echo "CSV:  ${CSV_PATH}  ($(du -sh ${CSV_PATH} | cut -f1))"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)